projects/power_distribution/src/bps_watcher.c

```diff
@@ -0,0 +1,16 @@
+#include "bps_watcher.h"
+#include "event_queue.h"
+#include "pd_events.h"
+
+StatusCode prv_bps_watcher_callback_handler(const CanMessage *msg, void *context,
+                                            CanAckStatus *ack_reply) {
+  uint8_t data = 0;
+  CAN_UNPACK_BPS_HEARTBEAT(msg, &data);
+  if (data) event_raise(POWER_DISTRIBUTION_STROBE_EVENT, 1);
```

Comment:
We also need to turn off to whatever's powered on aux, right? So raise a `POWER_DISTRIBUTION_POWER_SEQUENCE_EVENT_TURN_ON_EVERYTHING_AUX` as well (with a comment, bc it sounds like you're turning things on).

Also, I don't think a raw `if (data)` is recommended, better would be `if (data != 0)`, or even better to be self-documenting, `#define NO_FAULT 0` and then `if (data != NO_FAULT)`.